pkgs/development/python-modules/flet/default.nix

```
@@ -62,6 +63,11 @@ buildPythonPackage rec {
 
   doCheck = false;
 
+  fixupPhase = ''
+    wrapProgram $out/bin/flet \
+      --prefix PYTHONPATH : $PYTHONPATH
+  '';
```

```suggestion
makeWrapperArgs = [
  "--prefix PYTHONPATH : $PYTHONPATH"
];
```

However, I can't reproduce the import error because `bin/.flet-wrapped` already contains a line like this

```python
import sys;import site;import functools;sys.argv[0] = '/nix/store/6ynbyvrzxy5g7maa7w8xm5mhxz23kmvv-python3.11-flet-0.22.1/bin/flet';functools.reduce(lambda k, p: site.addsitedir(p, k), ['/nix/store/6ynbyvrzxy5g7maa7w8xm5mhxz23kmvv-python3.11-flet-0.22.1/lib/python3.11/site-packages','/nix/store/4akqnwj2agnmf7jpr2w782d76yaalzfs-python3.11-fastapi-0.110.2/lib/python3...
```

---

Try with any example from https://github.com/flet-dev/examples

---

It only happens when you actually open a example, --help works for me too.

---

Please try to find out why the additional wrapping is needed